.vsts-ci/linux.yml

```diff
@@ -60,33 +64,34 @@ stages:
       jobName: linux_build
       displayName: linux Build
 
-- stage: TestLinux
-  displayName: Test for Linux
+- stage: TestUbuntu
+  displayName: Test for Ubuntu
+  dependsOn: [BuildLinuxStage]
   jobs:
   - template: templates/nix-test.yml
     parameters:
-      name: Linux
+      name: Ubuntu
```

Comment:
i suppose eventually we should be using `ubuntu-latest` and provide a little more future proofing

Reply:
Let's open a separate PR for that.   I don't particularly like that because the people on build duty usually just decide to roll back if there is an actual version upgrade.